.github/workflows/e2e-component-tests-embedding-sdk-cross-version.yml

```diff
@@ -165,8 +165,12 @@ jobs:
 
       - name: Move tests from the release commit into the repo
         run: |
-          rm -rf ./e2e/support/helpers/*
-          mv ./sdk-release-e2e/e2e/support/helpers/* ./e2e/support/helpers
+          mv ./e2e/support/component-webpack.config.js .
```

Comment:
there are 2 things here. We used to assume we only needed `support/helpers` which was valid until got a reality check with https://github.com/metabase/metabase/pull/52878.

Now we shouldn't really rely on arbitrary directories, we should just copy over everything.

But, there's a catch. We just [merged a fix](https://github.com/metabase/metabase/pull/52800) to the release branch to fix this very test. So we need this webpack config file from the release branch.

That is why I move it somewhere else before nuking `e2e/support` and move it back after I copy `e2e/support` from the SDK release tag.